benchmark/scenarios/test/test_read_request_load.py

```diff
@@ -183,6 +183,7 @@ def test_scenario_throws_exception_when_already_started(self, _logger):
 
         self.assertRaises(RequestScenarioAlreadyStarted, s.start)
 
+
```

Comment:
:grey_question: I thought we did just 1 empty line between member functions, but lint appears to be happy so I'm happy. For consistency, it'd probably be better with only 1 newline. But that's better enforced by linters than reviewers, so no action required.